gui/default/syncthing/core/syncthingController.js

```diff
@@ -1192,6 +1192,61 @@ angular.module('syncthing.core')
             return '?';
         };
 
+        $scope.rdConnType = function(deviceID){
+            var conn = $scope.connections[deviceID];
+            if(!conn)
+                return "-1";
+                
+            if (conn.type.indexOf('relay') == 0)
```

Comment:
```suggestion
            if (conn.type.indexOf('relay') === 0)
```

For the sake of consistency.